1-js/07-object-properties/02-property-accessors/article.md

```diff
@@ -175,19 +175,19 @@ let user = {
 user.name = "Pete";
 alert(user.name); // Pete
 
-user.name = ""; // Name is too short...
+user.name = ""; // ...اسم خیلی کوتاه است
 ```
 
-So, the name is stored in `_name` property, and the access is done via getter and setter.
+پس اسم در ویژگی `_name` ذخیره شده است و دسترسی گرفتن توسط گیرنده و تنظیم‌کننده انجام می‌گیرد.
 
-Technically, external code is able to access the name directly by using `user._name`. But there is a widely known convention that properties starting with an underscore `"_"` are internal and should not be touched from outside the object.
+از لحاظ فنی، کد بیرونی می‌تواند به صورت مستقیم با استفاده از `user._name` به اسم دسترسی پیدا کند. اما یک قرارداد شناخته شده وجود دارد که ویژگی‌هایی که با یک زیرخط (underscore) `"_"` شروع می‌شوند، داخلی هستند و نباید بیرون از شیء به آن‌ها کاری داشت.
 
 
-## Using for compatibility
+## استفاده برای سازگاری
 
-One of the great uses of accessors is that they allow to take control over a "regular" data property at any moment by replacing it with a getter and a setter and tweak its behavior.
+یکی از بهترین کاربردهای دسترسی‌ها این است که آن‌ها به ما اجازه می‌دهند که در هر زمان یک ویژگی داده‌ای «معمولی» را از طریق جایگزین کردن آن با یک گیرنده و یک تنظیم‌کننده کنترل کنیم و رفتار آن را تغییر دهیم.
```

Comment:
Line 188 needs more attention

Reply:
The words "Getter" and "Setter" are well-known among software developers. you don't need to translate them.
```suggestion
یکی از بهترین کاربردهای اکسسرها(accessors) این است که آن‌ها به ما اجازه می‌دهند که در هر زمان یک ویژگی داده‌ای «معمولی» را از طریق جایگزین کردن آن با یک getter و یک setter کنترل کنیم و رفتار آن را تغییر دهیم.
```